tangelo/toolboxes/optimizers/tests/test_rotosolve.py

```diff
@@ -17,7 +17,8 @@
 import numpy as np
 
 from tangelo.linq import get_backend, Gate, Circuit
-from tangelo.toolboxes.optimizers.rotosolve import rotosolve
+from tangelo.toolboxes.optimizers.rotosolve import rotosolve, rotoselect
+from tangelo.toolboxes.operators.operators import QubitHamiltonian
```

Comment:
@alexfleury-sb Would you recommend here we use `QubitOperator` instead ? I believe we haven't followed through on `QubitHamiltonian`: we currently have a class that exists but is not really in use ? It seems here that what is used in the assert statement in the text is energy

Reply:
Yes, I think the behaviour of `QubitHamiltonian` and `QubitOperator` should be the same. We never used it to its full potential, and it can cause hiccups in some situations. This is why staying with `QubitOperator` is safer.

Reply:
In commit [3a4b907](https://github.com/goodchemistryco/Tangelo/pull/392/commits/3a4b907d3360008385c159d00df91b835277be29) I have changed the tests to use `QubitOperator`.